tests/css/CSSCombBearTest.py

```diff
@@ -0,0 +1,277 @@
+    CSSCombBear,
+    invalid_files=(),
+    valid_files=(test_file5,),
+    settings=settings)
+
+
+CSSCombBearCoafileTest2 = verify_local_bear(
+    CSSCombBear,
+    invalid_files=(test_file2, test_file3, test_file4),
+    valid_files=(),
+    settings=settings)
+
+
+CSSCombBearSemicolonTest = verify_local_bear(
+    CSSCombBear,
+    invalid_files=(test_file_always_semicolon,),
+    valid_files=(),
+    settings={"always-semicolon": "True"})
+
+
+CSSCombBearSpaceBefore = verify_local_bear(
+    CSSCombBear,
+    invalid_files=(),
+    valid_files=(test_file1,),
+    settings={"space-before-colon": " "})
+
+
+CSSCombBearSpaceAfter = verify_local_bear(
+    CSSCombBear,
+    invalid_files=(test_file2, test_file4),
+    valid_files=(),
+    settings={"space-after-colon": " "})
+
+
+CSSCombBearSpaceAfter2 = verify_local_bear(
+    CSSCombBear,
+    invalid_files=(test_file3, test_file4),
+    valid_files=(),
+    settings={"space-after-colon": " ",
+              "space-before-closing-brace": "\\n"})
+
+
+CSSCombBearSpaceAfter3 = verify_local_bear(
+    CSSCombBear,
+    invalid_files=(test_file3, test_file4, test_file2),
+    valid_files=(),
+    settings={"space-after-colon": " ",
+              "space-before-closing-brace": "\\n",
+              "strip-spaces": "True"})
+
+
+CSSCombBearSpaceAfter4 = verify_local_bear(
+    CSSCombBear,
+    invalid_files=(),
+    valid_files=(test_file_space_after_close_brace, test_file1),
+    settings={"space-after-colon": " ",
+              "space-before-closing-brace": "\\n",
+              "strip-spaces": "True",
+              "vendor-prefix-align": "True",
+              "quotes": "single",
+              "leading-zero": "False"})
+
+
+CSSCombBearSpaceAfter5 = verify_local_bear(
+    CSSCombBear,
+    invalid_files=(test_file3, test_file4, test_file2),
+    valid_files=(),
+    settings={"space-after-colon": " ",
+              "space-before-closing-brace": "\\n",
+              "strip-spaces": "True",
+              "vendor-prefix-align": "True",
+              "quotes": "single",
+              "leading-zero": "False",
+              "eof-newline": "True"})
+
+
+CSSCombBearSpaceAfter6 = verify_local_bear(
+    CSSCombBear,
+    invalid_files=(),
+    valid_files=(test_file_space_after_close_brace, test_file1),
+    settings={"space-after-colon": " ",
+              "space-before-closing-brace": "\\n",
+              "strip-spaces": "True",
+              "vendor-prefix-align": "True",
+              "quotes": "single",
+              "leading-zero": "False",
+              "eof-newline": "True",
+              "block-indent": "\\t"})
+
+
+CSSCombBearRulesetsTest = verify_local_bear(
+    CSSCombBear,
+    invalid_files=(test_file_remove_empty_rulesets,),
+    valid_files=(),
+    settings={"remove-empty-rulesets": True})
+
+
+CSSCombBearSemicolonTest2 = verify_local_bear(
+    CSSCombBear,
+    invalid_files=(test_file_add_semicolon,),
+    valid_files=(),
```

Comment:
i see youre missing tests. for example, here you need to give a valid file i think. and this is the case in many places, you need to have a valid file and an invalid file. @Asnelchristian could you help please?